### DISCUSSION TITLE
[supabase-py] Count select query

### DISCUSSION THREAD

**Discussion by smarcet** [supabase/supabase#1404] [Questions] [answered]
how do u perform a count select query on a field using python SDK
like stated on js doc

const { data, error, count } = await postgrest
  .from('cities')
  .select('name', { count: 'exact' })

regards

**Comment by steve-chavez** [comment, marked as answer]
I think the count interface is not yet implemented on `supabase-py`, you might want to open an issue there https://github.com/supabase/supabase-py

That being said, I think this other way works now:

```python
supabase.table("countries").select("count").execute()
```
